styx-scheduler-service/src/main/java/com/spotify/styx/StyxScheduler.java

```diff
@@ -590,12 +591,12 @@ private static DockerRunner createDockerRunner(
       LOG.info("Creating LocalDockerRunner");
       return closer.register(DockerRunner.local(scheduler, stateManager));
     } else {
-      final KubernetesClient kubernetes = closer.register(getKubernetesClient(config, id));
+      final NamespacedKubernetesClient kubernetes = closer.register(getKubernetesClient(config, id));
```

Comment:
The `inNamespace` method is only available on the `NamspacedKubernetesClient`. 

https://github.com/spotify/styx/blob/master/styx-scheduler-service/src/main/java/com/spotify/styx/docker/KubernetesDockerRunner.java#L104